packages/x-charts/src/internals/plugins/featurePlugins/useChartPolarAxis/coordinateTransformation.ts

```diff
@@ -0,0 +1,16 @@
+export const generateSvg2rotation =
+  (center: { cx: number; cy: number }) => (x: number, y: number) =>
+    Math.atan2(x - center.cx, center.cy - y);
+
+export const generateSvg2polar =
+  (center: { cx: number; cy: number }) =>
+  (x: number, y: number): [number, number] => {
+    const angle = Math.atan2(x - center.cx, center.cy - y);
+    return [Math.sqrt((x - center.cx) ** 2 + (center.cy - y) ** 2), angle];
+  };
+
+export const generatePolar2svg =
```

Comment:
Are we converting between cartesian to/from polar coordinates? Would it make sense to call these `polarToCartesian` or `cartesianToPolar` instead?

Reply:
I preferred svg over cartesian to avoid the confusion with the cartesian charts